bookstore/archive.py

```diff
@@ -139,12 +139,13 @@ def run_pre_save_hook(self, model, path, **kwargs):
         Parameters
         ----------
 
-        model : str
-            The type of file
+        model : dict
+            The type of file and its contents
         path : str
             The storage location
         """
         if model["type"] != "notebook":
+            self.log.info("Bookstore only archives notebooks.")
```

Comment:
Thinking about what this will look like in the logs, if someone is editing any file, it's going to show up as:

```
[I 16:46:40.790 NotebookApp] Bookstore only archives notebooks.
[I 16:48:40.790 NotebookApp] Bookstore only archives notebooks.
[I 16:50:40.790 NotebookApp] Bookstore only archives notebooks.
```

so I'd recommend setting this to `self.log.debug` and/or adding the filename to the message so this isn't useless amidst larger user logs (for operators to sift through).

Reply:
That's a really good point. I'll make it debug.

Reply:
I also added a line for specifying the file in question